libs/testnorge-common-command/src/main/java/no/nav/registre/testnorge/libs/common/command/CreatePersonCommand.java

```diff
@@ -17,6 +18,7 @@
     private final WebClient webClient;
     private final PersonDTO person;
     private final String accessToken;
+    private final String kilde;
```

Comment:
Prøvde å ikke ha `private final` foran kilde for å slippe å ha med null i CreatePersonCommand-konstruktøren i de andre consumerne, men fikk det ikke helt til å virke.

Reply:
Finnes det noe annet alternativ enn dette?
```
public CreatePersonCommand(WebClient webClient, PersonDTO person, String accessToken) {
this.webClient = webClient;
this.person = person;
this.accessToken = accessToken;
this.kilde = null;
}
```